src/Core/Query/index.ts

```diff
@@ -1 +1,3 @@
 export * as Query from "./Query";
+import { GetFacts } from "./GetFacts";
```

Comment:
Yes I changed the way we export the queries. Usually we define the sortkey and filter types in the domain.
But as they are mostly related to the query, I defined those types in the query file.
To avoid having to name it `GetFactsFilter` & `GetFactsSortKey`, I bundle them in the `GetFacts` namespace.
Now you can use `GetFacts` + `.Query` or `.Manifest` or `.Filter` or `.SortKey`. This seemed more clear to me.
Typescript already support exporting modules this way. But ESLint is not able to handle it yet, so that why we have this special import export fix. We already do it for some other modules. Not for all.